src/configs/commands.js

```diff
@@ -0,0 +1,10 @@
+const { ServerTime } = require("../useCases");
+
+const commands = [
```

Comment:
this `commands` array could be something like
```js
const commands = new Map();

commands.set(`${commandPrefix}${dayzTime}`, new ServerTime().execute);
```
and could be accessed by `commands.get(message.content.split(' ')[0])`, so finding a command will be easier